tests/newsModel.test.js

```diff
@@ -0,0 +1,23 @@
+const NewsModel = require("../src/newsModel")
+
+describe("NewsModel", ()=>{
+	let model;
+	beforeEach(()=>{
+		model = new NewsModel();
+	})
+
+	it("creates an instance of itself", ()=>{
+		expect(model).toBeInstanceOf(NewsModel)
+	})
+
+	it("has a getNews() method", ()=>{
+		expect(model.getNews().length).toBe(0)
+	})
+
+	it("has a updateNews() method", ()=>{
```

Comment:
Same as above. The test is actually testing if the updateNews() adds to the array. Maybe the test could be written as it("has a updateNews() method which adds information to the news array")